src/core/cuda_context.cpp

```diff
@@ -0,0 +1,48 @@
+#include "cuda_context.h"
+#include "cuda_device.h"
+
+
+
+using namespace CUDA;
+
+
+
+
+
+
```

Comment:
Missing argument comments and description is incomplete(does not mention qobject)